src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double secondLeg, double firstLeg) {
+        super(color);
+        this.secondLeg = secondLeg;
+        this.firstLeg = firstLeg;
+    }

```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for `secondLeg` and `firstLeg` and pass them on for better code readability.